robocop/checkers/spacing.py

```diff
@@ -193,17 +193,19 @@ class InconsistentUseOfTabsAndSpacesChecker(VisitorChecker, ModelVisitor):
     }
 
     def __init__(self, *args):
-        self.found = False
-        self.tabs = False
-        self.spaces = False
+        self.found, self.tabs, self.spaces = False, False, False
         super().__init__(*args)
 
+    def visit_File(self, node):  # noqa
+        self.found, self.tabs, self.spaces = False, False, False
```

Comment:
without that, if the issues was found in first file it would stop checking for all other files